cypress/support/step_definition/admin/administrateCtlToolSteps.spec.js

```diff
@@ -53,10 +53,10 @@ Then("I do not see external tool {string} in the tool selection", (toolName) =>
 
 Then("I see the deactivate checkbox is {string}", (value) => {
 	management.seeDeactivatedCheckBox(value);
```

Comment:
I don't understood this assertion here `management.seeDeactivatedCheckBox(value);` because 
- first there is no use of `value` in the method
- second name of the method does not corresponds to the method working i.e. what we are checking here what to see?

second `if-else` operation should be done in a separate  method and that method have to be used here for simplicity 

another thing, if you have some more cases then it makes sense to use `if-else if` otherwise `if-else` suffice.